main.py

```diff
@@ -4,14 +4,26 @@
 from src.datasets.dataset import Dataset
 from src.utils.dataset_helper import group_instances_across_frames
 from src.datasets.nuscenes.nuscenes_dataset import NuscenesDataset
+from src.datasets.waymo.waymo_dataset import WaymoDataset
 
 from src.utils.visualisation_helper import visualise_points_cloud
 
 
+def __create_dataset(dataset: str) -> Dataset:
+    if dataset == 'nuscenes':
+        return NuscenesDataset(version='v1.0-mini', dataroot='./temp/nuscenes')
+    elif dataset == 'waymo':
+        return WaymoDataset(dataset_root='./temp/open-waymo')
+    else:
+        raise Exception(f"Unknown dataset {dataset}")
+
+
 def main():
-    dataset: Dataset = NuscenesDataset(version='v1.0-mini', dataroot='./temp/nuscenes')
+    dataset = __create_dataset('waymo')
 
-    scene_id = '0'
+    print('Scenes:', dataset.scenes)
+
+    scene_id = 'training_segment-10023947602400723454_1120_000_1140_000_with_camera_labels'
```

Comment:
is that waymo specific? but it's in main?

Reply:
`main` is just a playground for us to test: it works faster than `patch_scene` and has hard-coded values for ids.

Reply:
ok!